docs/diagrams/SessionClassUML.puml

```diff
@@ -0,0 +1,53 @@
+@startuml Session
+skinparam classAttributeIconSize 0
+
+class Session {
+  + getSessionNumber(): SessionNumber
+  + getStudents(): SessionStudents
+  + getSessionRemark(): SessionRemark
+  + isSameSession(Session): boolean
+  + getSessionInfo(): String
+  + hashCode(): int
+  + toString(): String
+}
+
+class SessionNumber {
+  + sessionNumber: String
+  + <u>MESSAGE_CONSTRAINTS: String
+  + <u>VALIDATION_REGEX: String
+  + <u>isValidSessionNumber(String): boolean
+  + hashCode(): int
+  + toString(): String
+}
+
+class SessionStudents {
+  + students: List<Person>
+  + <u>MESSAGE_CONSTRAINTS: String
+  + <u>VALIDATION_REGEX: String
+  + addStudent(Person)
+  + removeStudent(Person)
+  + getStudents(): SessionStudents
+  + toStudentNames(): String
+  + contains(Person): boolean
+  + hashCode(): int
+  + toString(): String
+}
+
+class SessionRemark {
+  + sessionRemark: String
+  + <u>MESSAGE_CONSTRAINTS: String
+  + <u>VALIDATION_REGEX: String
+  + <u>isValidSessionRemark(String): boolean
+  + hashCode(): int
+  + toString(): String
+}
```

Comment:
Is toString() and hashCode() necessary? Omit less important details and repetitive details.

Reply:
Will do! Thanks for the suggestion